### ISSUE TITLE
Difficult to tell which tab is focused

### ISSUE THREAD

**Issue by ryanprior** [Zonnev/elementaryos-firefox-theme#54]
If you open about 30 tabs, it becomes impossible to spot which one is focused by glancing at the tab bar.

Firefox offers a "tab line" which puts a colorful line above the focused tab, but this style disables it. It could also be helpful to choose a distinct background gradient for the focused tab.

Would you welcome a PR exploring some options to make the focused tab easy to identify?

**Comment by Pliavi** [comment]
I was thinking about it yesterday in my third days using this theme, i got the same difficult.
And seeing the Terminal tabs as it's an official/curated app, the tabs looks brighter with border to show whats is in use and darken a bit the non focused ones.

So maybe following this standard may improve this, but I like the tabline ideia, as even the oficial tabs theme is hard to follow with the eyes, and worse in dark mode (even it's not official and got only with elementary-tweaks).

Some examples that can improve visibility here: 
![image](https://user-images.githubusercontent.com/5985423/70629071-1bcda700-1c08-11ea-879e-44d342efa511.png)

 1. The main 'problem', the tabs in Terminal has this inset white shadow that make it a bit brighter, and differentiate the focused from the others.
And the focused in Terminal the gradient goes from bright to dark (top-bottom), and inverse when not. This themes do the same to non focused, but the focused one has only one color.
2. The rest of the app (the top bar) have the same light effect and the it's even brighter, making it shows segregate it from the tabs (and make the focused tab in the almost same level)
The Terminal most dark color is #3c3c3c, in the theme is #363636 and the most bright in Terminal is #3e3e3e, and theme #3b3b3b, it's relatively darker.
3. I would say to thin and darkenize the divisor, but I think it will not make too much difference.

I would like to see your idea in practice @ryanprior , and help if it's needed :grin:

**Comment by ryanprior** [comment]
![Screenshot from 2019-12-11 09 53 28](https://user-images.githubusercontent.com/4389854/70637021-271ad580-1bfc-11ea-8fa5-2311b59887d1.png)

Here's a view of the modification I've been using lately. It helps me a lot with my own Firefox usage and if I knew how to try out a similar thing in my terminal app I would give it a shot, but I haven't done any GTK styling before.

I typically have 1-2 terminal tabs open, at most maybe 6, and among that many it's not hard for me to pick out the focused one. For a person with vision impairment, or for somebody who has more tabs open where they're monitoring long-running async processes or similar, finding the focused tab at a glance gets harder.

Firefox is harder in two ways: first, when you have a lot of tabs open (like in my screen shot) it gives you narrow tabs which means lots of options for which one might be focused. Second, the tabs are made visually more distinct by the inclusion of favicons, which is great for finding which specific tabs are related to elementary OS or to GitHub etc, but makes it harder to spot the little "x" widget on the focused tab.

**Comment by Zonnev** [comment]
Thank you all for your extensive input! It made me adjust the darkmode to suit your needs a bit more. Install the new userChrome.

**Comment by Pliavi** [comment]
Wow! that was fast! :open_mouth: 
And you did a very good work here, thanks! :heart_eyes: 

@ryanprior you version (and the customCSS in the page) remembers a lot of Steam :thinking: hehe

**Comment by ryanprior** [comment]
Great work @Zonnev, this is a big improvement.
